requirements-dev.txt

```diff
@@ -17,6 +17,4 @@ pytest-tornasync
 pytest-twisted
 types-requests
 twisted
-aiosqlite
-aiomysql
-asyncpg
+sqlalchemy
```

Comment:
There is already the `sqlalchemy` package dependency in `requirements.txt` file, so it should not be here.

Reply:
Right, wiil remove from `requirements.txt`. Great thanks for your suggestions.